### ISSUE TITLE
App crashes when user is not assigned to a group

### ISSUE THREAD

**Issue by alpacawool** [Todo-App-Codepath/todoApp#33]
When signing in without a group, the app crashes. Once the user is assigned a group and app data is cleared, the app works as intended.

**Comment by jamiemott** [comment]
Yes, we still need to decide how to do this. Maybe a new group is made when they sign up without a group? Then they would be the only member until new people are invited.
